test/e2e/storage/testsuites/provisioning.go

```diff
@@ -151,7 +151,7 @@ func (p *provisioningTestResource) setupResource(driver drivers.TestDriver, patt
 				framework.Skipf("Driver %q does not define Dynamic Provision StorageClass - skipping", driver.GetDriverInfo().Name)
 			}
 			p.driver = driver
-			p.claimSize = "2Gi"
+			p.claimSize = "5Gi"
```

Comment:
@mkimuram This hard-coded size doesn't work for me: my test setup provisions in RAM and can't support such large volumes. This needs to be configurable.

I can work on a PR for this, but let's clarify first what the role of `framework.VolumeTestConfig` is in `testsuites`.

Speaking of volume size, the subpath tests fail for the same reason. But I haven't figured out yet where that PVC gets created. Can you point me in the right direction?

Reply:
@pohly 

This value was changed to 5Gi due to the restriction of current version of csi gce driver.
Once @davidz627 push the latest version of csi gce driver, then we will be able to change it to smaller value.

Also, we will be able to tune it by defining something like minimum volume size in each driver and set the size here.

> But I haven't figured out yet where that PVC gets created. Can you point me in the right direction?

It depends on the testsuites but it should be created either in
1. Each tests ([Inside ```It()```](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/provisioning.go#L215)) (ex. provisioning, volumeMode testsuites)
2. [setupResource](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/base.go#L137) (ex. subPath, volumes, volumeIO testsuites)

It is different because 
 - if the scope of the test includes the creation of a resource, it should be inside ```It()```,
 - if the scope of the test doesn't include the creation of a resource, it should be in ```setupResource``` to avoid duplicated logic in all tests. (Most of tests will be able to use genericVolumeTestResource as is.)

Reply:
I agree on making PVC size configurable.  For example, Google Filestore volumes have a min size of 1TB